eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+  Artifacts: []
+  ServiceDirectory: not-specified
+  RunUnitTests: false
+
+stages:
+  - stage: Build
+    jobs:
+      - template: ../jobs/archetype-sdk-client.yml
+        parameters:
+          ServiceDirectory: ${{ parameters.ServiceDirectory }}
+          RunUnitTests: ${{ parameters.RunUnitTests }}
+
+      - template: ../jobs/archetype-sdk-integration.yml
+        parameters:
+          PackagePath: "sdk/cosmosdb/cosmos/"
+          PackageName: "@azure/cosmos"
+          Toolchain: rush
+          Matrix:
+            Windows_Node8:
+              OSVmImage: "vs2017-win2016"
```

Comment:
we are updating OSVMImage to 'windows-2019', so it would be good to go ahead and do that here.